api/catalog/api/views/audio_views.py

```diff
@@ -44,7 +45,7 @@ class AudioViewSet(MediaViewSet):
 
     model_class = Audio
     query_serializer_class = AudioSearchRequestSerializer
-    default_index = "audio"
+    default_index = settings.MEDIA_INDEX_MAPPING["audio"]
```

Comment:
Would it be better to use `AUDIO_TYPE` constant here (and `IMAGE_TYPE` in the next file)?

Reply:
Makes sense, updated in 5749ee4.